### PR TITLE
(PLANNED FOR 8/5) Release go 3.34.0

### PR DESCRIPTION
**Pull request by iamemilio**
Prepares the Go agent docs for the 3.34.0 release.

Release planned for tomorrow 4PM EST.

### CODE REVIEW

File: src/content/docs/logs/logs-context/configure-logs-context-go.mdx [hunk 1]
```diff
@@ -22,6 +22,65 @@ With our Go language <InlinePopover type="apm" /> agent, you can get <DNT>**logs
 Got lots of Go logs? Check out our [tutorial on how to optimize and manage them](/docs/tutorial-large-logs/get-started-managing-large-logs/). 
 </Callout>
 
+## Supported Logging Libraries
```
Comment (ally-sassman) [inline src/content/docs/logs/logs-context/configure-logs-context-go.mdx:25]:
```suggestion
## Supported logging libraries [#supported-libraries]
```

File: src/content/docs/logs/logs-context/configure-logs-context-go.mdx [hunk 2]
```diff
@@ -22,6 +22,65 @@ With our Go language <InlinePopover type="apm" /> agent, you can get <DNT>**logs
...
+
+<table>
+  <thead>
+    <tr>
+      <th>
+        Library
+      </th>
+
+      <th>
+        Supports Attribute Collection
```
Comment (ally-sassman) [inline src/content/docs/logs/logs-context/configure-logs-context-go.mdx:35]:
```suggestion
        Supported attribute collection
```

File: src/content/docs/logs/logs-context/configure-logs-context-go.mdx [hunk 3]
```diff
@@ -22,6 +22,65 @@ With our Go language <InlinePopover type="apm" /> agent, you can get <DNT>**logs
...
+      </th>
+
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        [log](https://github.com/newrelic/go-agent/tree/master/v3/integrations/logcontext-v2/logWriter)
```
Comment (ally-sassman) [inline src/content/docs/logs/logs-context/configure-logs-context-go.mdx:43]:
```suggestion
        [`logWriter`](https://github.com/newrelic/go-agent/tree/master/v3/integrations/logcontext-v2/logWriter)
```

File: src/content/docs/logs/logs-context/configure-logs-context-go.mdx [hunk 4]
```diff
@@ -22,6 +22,65 @@ With our Go language <InlinePopover type="apm" /> agent, you can get <DNT>**logs
...
+      </td>
+      <td>
+        No
+      </td>
+    </tr>
+    <tr>
+      <td>
+        [slog](https://github.com/newrelic/go-agent/tree/master/v3/integrations/logcontext-v2/nrslog)
```
Comment (ally-sassman) [inline src/content/docs/logs/logs-context/configure-logs-context-go.mdx:51]:
```suggestion
        [`slog`](https://github.com/newrelic/go-agent/tree/master/v3/integrations/logcontext-v2/nrslog)
```

File: src/content/docs/logs/logs-context/configure-logs-context-go.mdx [hunk 5]
```diff
@@ -22,6 +22,65 @@ With our Go language <InlinePopover type="apm" /> agent, you can get <DNT>**logs
...
+      </td>
+      <td>
+        Yes
+      </td>
+    </tr>
+    <tr>
+      <td>
+        [Logrus](https://github.com/newrelic/go-agent/tree/master/v3/integrations/logcontext-v2/nrlogrus)
```
Comment (ally-sassman) [inline src/content/docs/logs/logs-context/configure-logs-context-go.mdx:59]:
```suggestion
        [`nrlogrus`](https://github.com/newrelic/go-agent/tree/master/v3/integrations/logcontext-v2/nrlogrus)
```

File: src/content/docs/logs/logs-context/configure-logs-context-go.mdx [hunk 6]
```diff
@@ -22,6 +22,65 @@ With our Go language <InlinePopover type="apm" /> agent, you can get <DNT>**logs
...
+      </td>
+      <td>
+        Yes
+      </td>
+    </tr>
+    <tr>
+      <td>
+        [Zap](https://github.com/newrelic/go-agent/tree/master/v3/integrations/logcontext-v2/nrzap)
```
Comment (ally-sassman) [inline src/content/docs/logs/logs-context/configure-logs-context-go.mdx:67]:
```suggestion
        [nrzap](https://github.com/newrelic/go-agent/tree/master/v3/integrations/logcontext-v2/nrzap)
```

File: src/content/docs/logs/logs-context/configure-logs-context-go.mdx [hunk 7]
```diff
@@ -22,6 +22,65 @@ With our Go language <InlinePopover type="apm" /> agent, you can get <DNT>**logs
...
+      </td>
+      <td>
+        Yes
+      </td>
+    </tr>
+    <tr>
+      <td>
+        [ZeroLog](https://github.com/newrelic/go-agent/tree/master/v3/integrations/logcontext-v2/zerologWriter)
```
Comment (ally-sassman) [inline src/content/docs/logs/logs-context/configure-logs-context-go.mdx:75]:
```suggestion
        [`zerolog-writer`](https://github.com/newrelic/go-agent/tree/master/v3/integrations/logcontext-v2/zerologWriter)
```

File: src/content/docs/release-notes/agent-release-notes/go-release-notes/go-agent-3-34-0.mdx [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+We recommend updating to the latest agent version as soon as it's available. If your organization has established practices that prevent you from upgrading to the latest version, ensure that your agents are regularly updated to a version that's at most 90 days old. Read more about [keeping your agent up to date](/docs/new-relic-solutions/new-relic-one/install-configure/update-new-relic-agent/).
+</Callout>
+
+## 3.34.0
+### Added
+  - logcontext-v2/nrlogrus can now collect user attributes
+  - logcontext-v2/nrslog can now collect user attributes
+  - use slog to manage Go agent logs with the new nrslog library
+  - use zerolog go manage Go agent logs with the new nrzerolog library
+  - support for `RegisterTLSConfig` in nrmysql
```
Comment (ally-sassman) [inline src/content/docs/release-notes/agent-release-notes/go-release-notes/go-agent-3-34-0.mdx:21]:
```suggestion
  - `logcontext-v2/nrlogrus` can now collect user attributes
  - `logcontext-v2/nrslog` can now collect user attributes
  - Use `slog` to manage Go agent logs with the new `nrslog` library
  - Use `zerolog` go manage Go agent logs with the new `nrzerolog` library
  - Support for `RegisterTLSConfig` in `nrmysql`
```

File: src/content/docs/release-notes/agent-release-notes/go-release-notes/go-agent-3-34-0.mdx [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+### Fixed
+  - logcontext-v2/nrlogrus will still print user logs without isses if the Go agent has already been shut down.
+  - switched protobuff to google.golang.org/protobuff in modfile
```
Comment (ally-sassman) [inline src/content/docs/release-notes/agent-release-notes/go-release-notes/go-agent-3-34-0.mdx:24]:
```suggestion
  - `logcontext-v2/nrlogrus` will still print user logs without issues if the Go agent has already been shut down
  - Switched `protobuff` to `google.golang.org/protobuff` in `modfile`
```

**Comment by ally-sassman** [comment]
@iamemilio Please see my suggestions. Let's keep the naming conventions the same for all the libraries (i.e., list them all by their code name).

**Review by ally-sassman — CHANGES_REQUESTED** [review]
See my comments/suggestions

**Comment by iamemilio** [comment]
Hi @ally-sassman we are planning to defer the release to monday august 5th, just so you know.

**Comment by iamemilio** [comment]
> See my comments/suggestions

Addressed! How does it look?

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**